cypress/src/plugins/index.ts

```diff
@@ -22,8 +22,9 @@ import coverageTask from '@cypress/code-coverage/task';
 const setupPluginsAndConfig: Cypress.PluginConfig = (on, config) => {
     coverageTask(on, config);
 
-    if (process.env.MANAGER_IP) {
-        config.baseUrl = `http://${process.env.MANAGER_IP}`;
+    const { MANAGER_IP, MANAGER_PROTOCOL } = process.env;
+    if (MANAGER_IP) {
+        config.baseUrl = `${MANAGER_PROTOCOL ?? 'http'}://${MANAGER_IP}`;
```

Comment:
Is plain HTTP still going to be supported?

Reply:
SSL is opt-out, so I think yes :slightly_smiling_face:

Reply:
Yes, it will still be possible to configure manager without SSL support.
From Lukasz M.: `manager: security: ssl_enabled: false` in `config.yaml`